modules/daicard/src/App.js

```diff
@@ -271,15 +271,15 @@ class App extends React.Component {
         })`,
       );
     } else if (useWalletConnext) {
-      let rpc = {};
-      rpc[network.chainId] = urls.ethProviderUrl;
-      const channelProvider = new WalletConnectChannelProvider();
+      const rpc = {
+        [network.chainId]: urls.ethProviderUrl
+      }
+      const channelProvider = new WalletConnectChannelProvider({
+        rpc,
+        chainId: network.chainId,
+      });
```

Comment:
Why not use this rpc value directly in the one place it's needed. Looks like we aren't gonna need that `rpc` var again later so I vote we remove it.